internal/plugin/connectors/tcp/template/connector.go

```diff
@@ -0,0 +1,22 @@
+package template
+
+import (
+	"net"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+)
+
+type SingleUseConnector struct {
+	logger log.Logger
+}
+
+func (connector *SingleUseConnector) Connect(
+	clientConn net.Conn,
+	credentialValuesByID connector.CredentialValuesByID,
+) (net.Conn, error) {
+	// TODO: add logic according to
+	//  https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/README.md#tcp-connector
+
+	return nil, nil
```

Comment:
This skeleton should have a bit more stuff probably (it should connect to the backend and return a `net.Conn`).

Reply:
how can i connect to a backend without knowing what it is? 
Or do you think I should add it as a comment?

whenever i linked to a doc i didn't add more details so it won't be duplicated. wdyt? @sgnn7

Reply:
You can add an example of it trying to parse "host" and "port" from `credentialValuesByID`

Reply:
i couldn't find a generic way to write that. I updated the function description to better describe its process